prometheus/prometheus_targets.yml

```diff
@@ -56,7 +56,7 @@
 - labels:
     job: codeinsights-db
   targets:
-    - codeinsights-db:9187
+    - codeinsights-db-exporter:9187
```

Comment:
Just double checking that this got manual eyes

Reply:
Yeah I don't really know what this is, but I don't think it's coming from the release tool

Reply:
Cool. Seems to be in the right commit either way.